package.json

```diff
@@ -103,6 +103,9 @@
   "devEngines": {
     "node": "8.x || 9.x"
   },
+  "jest": {
+    "testRegex": "/scripts/jest/dont-run-jest-directly\\.js$"
```

Comment:
Couldn't figure out any way to run something once before any tests are run, so I settled for making it think there's just one test.

Reply:
smart